source/v2/greenlanes-openapi.yaml

```diff
@@ -530,154 +520,260 @@ components:
+          id: "20098001"
+          type: measure
+          attributes:
+            effective_start_date: "2021-01-01T00:00:00.000Z"
+            effective_end_date: null
+          relationships:
+            goods_nomenclature:
+              data:
+                id: "30256"
+                type: goods_nomenclature
+            measure_type:
+              data:
+                id: "410"
+                type: measure_type
+            footnotes:
+              data:
+                - id: CD624
+                  type: footnote
+
+    MeasureType:
+      description: A referenced MeasureType object
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for this measure
+        description:
+          type: string
+          description: The description for the measure type
+        measure_type_series_description:
+          type: string
+          description: The description of the measure type series
+        validity_start_date:
+          type: string
+          description: The validity start date of the measure type
+          format: date-time
+        validity_end_date:
+          type: string
+          description: The validity end date of the measure type
+          format: date-time
+          nullable: true
+        measure_type_series_id:
+          type: string
+          description: A single char representing measure type series.
+          format: byte
+        trade_movement_code:
+          type: integer
+          description: The code of trade movement.
+      example:
+        data:
+          id: "110"
+          type: measure_type
+          attributes:
+            description: "Supplementary unit import"
+            measure_type_series_description: "Supplementary unit"
+            validity_start_date: "1972-01-01T00:00:00.000Z"
+            validity_end_date: null
+            measure_type_series_id: "O"
+            trade_movement_code: 0
+
+    Footnote:
+      description: A referenced Footnote object
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the footnote
+        code:
+          type: string
+          description: The `code` for the footnote
+        description:
+          type: string
+          description: The `description` for the footnote
+      example:
+        data:
+          id: TN701
+          type: footnote
+          attributes:
+            code: TN701
+            description: "According to  the Council Regulation (EU) No 692/2014
+              (OJ L183, p. 9) it shall be prohibited to import into European Union
+              goods originating in Crimea or Sevastopol.\n..."
+
+    ReferencedGoodsNomenclature:
+      description: A referenced GoodsNomenclature which a measure is against
+      type: object
+      properties:
+        id:
+          type: string
+          description: The unique id for this revision of this Goods Nomenclature
```

Comment:
There is a lot of redundancy of attr. specifying 
`of this Goods Nomenclature` , `of the goods nomenclature`, ...
Maybe we can remove those and make it more readable.